hack/packet-provision.sh

```diff
@@ -17,6 +17,7 @@ if [ "$TF_VAR_packet_project_id" == "" ]; then
 fi
 
 export TF_VAR_id=${ID:-$(uuidgen | cut -c1-8)}
+export TF_VAR_tag=${NODE_NAME:-usermachine}
```

Comment:
If `NODENAME` is unset (for everyone) then this become `usermachine` (for everyone). Yes? Do we want to add a prefix that help with identity?

Reply:
Yes, that's the idea. `NODE_NAME` is a variable always set by jenkins. Essentially this is a failsafe aimed at CI system so I wasn't creating anything  for users.